src/scores/processing/isoreg_impl.py

```diff
@@ -503,10 +503,13 @@ def _contiguous_mean_ir(
 ) -> np.ndarray:
     """
     Performs classical (i.e. for mean functional) contiguous quantile IR on tidied data x, y.
-    Uses sklearn implementation rather than supplying the mean solver function to `_contiguous_ir`,
-    as it is about 4 times faster (since it is optimised for mean).
+    Refactored to use scipy instead of scikit learn.
+
+    IMPORTANT NOTE: the y array MUST be sorted in descending order before being passed to the function. This
+    implicitly handled by the _tidy_ir_inputs function before calling this function.
+    To make explicit could include y = np.sort(y)[::-1]
```

Comment:
Re. the sentence "To make explicit could include y = np.sort(y)[::-1]", could the grammar be improved to make this clearer? (Or is it sufficiently clear as is?)

Reply:
This looks to be tidied up in the latest version.